Src/cmor_axes.c

```diff
@@ -34,7 +34,18 @@ int cmor_calendar_c2i(char *calendar, cdCalenType * ical)
         *ical = cdJulian;
     else if (strcmp(calendar, "none") == 0)
         *ical = cdClim;
-    else {
+    else if (strcmp(calendar, "utc") == 0
+            || strcmp(calendar, "tai") == 0
+            || strcmp(calendar, "all_leap") == 0
+            || strcmp(calendar, "366_day") == 0) {
+        cmor_handle_error_variadic(
+            "A %s calendar is not currently an option for MIP results."
```

Comment:
How about "A %s calendar is an invalid selection for Model Intercomparison Project (MIP) data."

Reply:
Can we also augment the function info with:

```python
/************************************************************************/
/*                         cmor_calendar_c2i()                          */
/* This function recognizes all valid calendars in the CF 1.12 release */
/************************************************************************/